api/src/main/java/org/openmrs/module/aijar/AijarConstants.java

```diff
@@ -38,4 +38,211 @@
     public static final String HSD_TB_NUMBER = "d1cda288-4853-4450-afbc-76bd4e65ea70";
     
     public static final String DISTRICT_TB_NUMBER = "67e9ec2f-4c72-408b-8122-3706909d77ec";
+
+    public static final String UIC_GENERATOR_QUERY =  "insert into patient_identifier (patient_id,identifier_type,date_created,preferred,location_id,creator ,changed_by,voided_by,identifier,uuid) \n"
+            + "select patient_id,(select patient_identifier_type_id from patient_identifier_type where uuid='877169c4-92c6-4cc9-bf45-1ab95faea242' )as identifier,NOW() AS date_created,0 as preffered,NULL,1 AS creator,null as changed_by,null as voided_by, concat(left(country,1),'-',mid(birthdate,6,2), mid(birthdate,3,2),'-', if (gender='M', 1, 2) , '-',\n"
```

Comment:
the mid() function have you considered that this was introduced in 5.6 and majority of the systems out there use 5.5 mysql version.